core/templates/dev/head/pages/collection_editor/editor_tab/collection_node_creator_directive.html

```diff
@@ -1,10 +1,10 @@
 <h2>Add Exploration</h2>
 <form class="form-inline" ng-submit="createNewExploration()">
-  <input class="form-control" ng-model="newExplorationTitle" type="text" placeholder="New exploration title">
+  <input class="form-control" ng-model="newExplorationTitle" type="text" placeholder="New Exploration title">
```

Comment:
Why capitalize "Exploration" here but not below? (Probably want to uncapitalize this one.)